lib/repl.js

```diff
@@ -1058,7 +1058,7 @@ REPLServer.prototype.defineCommand = function(keyword, cmd) {
     cmd = {action: cmd};
   } else if (typeof cmd.action !== 'function') {
     throw new errors.TypeError('ERR_INVALID_ARG_TYPE',
-      'action', 'function', cmd.action);
+                               'action', 'function', cmd.action);
```

Comment:
I have not noticed that as the predominant style, although I would welcome that change personally. I'll do that here, if no one objects.